test/integration/cluster-init.sh

```diff
@@ -12,12 +12,18 @@ suite_dir="${OS_ROOT}/test/integration/cluster-init/"
 tempdir="${BASETMPDIR}/cluster-init"
 mkdir -p "${tempdir}"
 cp -a "${suite_dir}"/* "${tempdir}"
-actual="${tempdir}/input"
-expected="${suite_dir}/expected"
 
 os::test::junit::declare_suite_start "integration/cluster-init"
 
-os::cmd::expect_success "cluster-init -cluster-name=newCluster -release-repo=${actual} -create-pr=false"
-os::integration::compare "${actual}" "${expected}"
+# test the create scenario
+actual_create="${tempdir}/create/input"
+expected_create="${suite_dir}/create/expected"
+os::cmd::expect_success "cluster-init -cluster-name=newCluster -release-repo=${actual_create} -create-pr=false"
+os::integration::compare "${actual_create}" "${expected_create}"
+# test the update scenario
+actual_update="${tempdir}/update/input"
+expected_update="${suite_dir}/update/expected"
+os::cmd::expect_success "cluster-init -cluster-name=existingCluster -release-repo=${actual_update} -update=true -create-pr=false"
```

Comment:
Why does this require a separate input and output? When would we expect it to actually change something from the above?

Reply:
I actually went in and made some fake changes to the new input (for example adding extra args to the commands in the jobs) to verify that the update actually changed it back to the expected spec. I didn't want the create scenario to have those silly changes in there, so I opted to keep the directories separate. 

As for the output, it will be different in the update scenario because we aren't adding a new cluster, but only updating the existing one.

I thought about just chaining these tests, so do a create, and then do an update to the output of the create. However, I wanted the results to be independent. If I chained them then the update would always fail when the create did.